Tools/ardupilotwaf/boards.py

```diff
@@ -462,6 +463,7 @@ def configure_env(self, cfg, env):
 
         env.PX4_BOOTLOADER_NAME = self.bootloader_name
         env.PX4_BOARD_NAME = self.board_name
+        env.PX4_BOARD_RC = self.board_rc
```

Comment:
could you add a comment explaining what board_rc is for? radio control? release candidate?